snowfakery/template_funcs.py

```diff
@@ -68,6 +66,7 @@ def render_boolean(context: PluginContext, value: FieldDefinition) -> bool:
 class StandardFuncs(SnowfakeryPlugin):
     class Functions:
         int = int
+        _faker_for_dates = Faker(use_weighting=False)  # use ONLY for random_dates
```

Comment:
This makes me wonder _why_ I shouldn't use it for something else

Reply:
5e9be1e1a2bafbf654f2ff3d441727363f540c5a